src/core/base/root.ts

```diff
@@ -51,17 +45,16 @@ export class UIPRoot extends ESLBaseElement {
   }
 
   protected connectedCallback() {
-    this.applyRewriteQuery(this.rewriteMode);
     super.connectedCallback();
     this.theme = String(this.theme);
+    this.settings = String(this.settings);
+    this.editor = String(this.editor);
```

Comment:
Please, anybody, explain me, why at all we have to do this assignments like 
```
this.theme = String(this.theme);
this.settings = String(this.settings);
```
I believe that it should work by default as it's default behavior of `attr()` decorator